### PR TITLE
uploading computer vision files

### PR DESCRIPTION
**Pull request by hawc2**
Hey @svmelton and @nabsiddiqui, this is my first lesson I'm publishing on Jekyll, so bear with me if there are any issues. Can you both sign off on this two-part lesson? Thanks!

### Checklist

- [ ] Assign yourself in the "Assignees" menu
- [ ] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [ ] Add the appropriate "Label"
- [ ] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [ ] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

File: _data/ph_authors.yml [hunk 1]
```diff
@@ -265,14 +265,15 @@
 
 
 - name: Daniel van Strien
+  orcid: 0000-0003-1684-6556
   team: false
   bio:
     en: |
-      Daniel van Strien is a Library Science student at City University, London.
```
Comment (svmelton) [inline _data/ph_authors.yml:271]: Extremely, extremely minor, but there's inconsistency in periods after the bios here

File: en/lessons/computer-vision-deep-learning-pt1.md [hunk 2]
```diff
@@ -0,0 +1,580 @@
+---
+title: "Computer Vision for the Humanities: An Introduction to Deep Learning for Image Classification (Part 1)"
+collection: lessons
+layout: lesson
+slug: computer-vision-deep-learning-pt1
+date: 2021-01-26
```
Comment (svmelton) [inline en/lessons/computer-vision-deep-learning-pt1.md]: You'll want to update the publication date for both lessons (and remove the translation date)

Comment (hawc2) [reply en/lessons/computer-vision-deep-learning-pt1.md]: thanks! i made those updates. just not sure why the build broke

**Comment by hawc2** [comment]
@ZoeLeBlanc could you take a look at this failed build? 

I've tried alot of testing to isolate what the problem is. It definitely seems to be an issue with both part 1 and 2 of this lesson, and I can't find any problem with the yaml header. @anisa-hawes did some work with rendering a couple tables, but even when I remove those it still throws the same error locally and on github pages. maybe it's a problem with liquid syntax or something?

Here's the main error message: `undefined method `[]' for nil:NilClass (NoMethodError)`

**Comment by hawc2** [comment]
Nevermind everyone! Looks like I got it fixed. @svmelton and @nabsiddiqui can you sign off on these lessons when you think they're good to go? Let's publish soon

**Comment by hawc2** [comment]
@svmelton I just noticed the gallery images aren't greyscale. Let me know if you or @nabsiddiqui find any other problems, but I'm making a note to self to fix that before we merge

**Comment by nabsiddiqui** [comment]
@hawc2 All looks good to me.

**Review by nabsiddiqui — APPROVED** [review]

**Comment by davanstrien** [comment]
@hawc2 I've created grayscale versions of the gallery images (attached below). 

computer-vision-deep-learning-pt1
computer-vision-deep-learning-pt2

**Comment by hawc2** [comment]
thanks! I'll try to get this published tomorrow

**Pull request merged**